From cf0f8888b11a2bc2be174fe65564684e1235c7d2 Mon Sep 17 00:00:00 2001
From: Armando Uribe De Leon <x0095078@ti.com>
Date: Thu, 6 May 2010 13:24:39 -0500
Subject: [PATCH 119/135] DSPBRIDGE: Remove checkpatch errors: externs

This patche is intented to remove the
checkpatch errors due to the definition
of externs in c source files.

Signed-off-by: Armando Uribe De Leon <x0095078@ti.com>
---
 arch/arm/plat-omap/include/dspbridge/dbll.h  |    3 +++
 arch/arm/plat-omap/include/dspbridge/dmm.h   |    5 +++++
 arch/arm/plat-omap/include/dspbridge/io_sm.h |   10 ++++++++++
 arch/arm/plat-omap/include/dspbridge/node.h  |    3 +++
 arch/arm/plat-omap/include/dspbridge/proc.h  |    2 ++
 drivers/dsp/bridge/pmgr/dbll.c               |    1 -
 drivers/dsp/bridge/pmgr/dmm.c                |    3 ---
 drivers/dsp/bridge/rmgr/drv.c                |    1 -
 drivers/dsp/bridge/rmgr/node.c               |    5 +----
 drivers/dsp/bridge/rmgr/proc.c               |    2 --
 drivers/dsp/bridge/wmd/_tiomap.h             |    6 ++++++
 drivers/dsp/bridge/wmd/_tiomap_pwr.h         |    6 ++++++
 drivers/dsp/bridge/wmd/io_sm.c               |   11 -----------
 drivers/dsp/bridge/wmd/mmu_fault.h           |    2 ++
 drivers/dsp/bridge/wmd/tiomap3430.c          |    6 ------
 drivers/dsp/bridge/wmd/tiomap3430_pwr.c      |    5 -----
 drivers/dsp/bridge/wmd/ue_deh.c              |    1 -
 17 files changed, 38 insertions(+), 34 deletions(-)

diff --git a/arch/arm/plat-omap/include/dspbridge/dbll.h b/arch/arm/plat-omap/include/dspbridge/dbll.h
index 01c4647..9c4c448 100644
--- a/arch/arm/plat-omap/include/dspbridge/dbll.h
+++ b/arch/arm/plat-omap/include/dspbridge/dbll.h
@@ -23,6 +23,8 @@
 #include <dspbridge/dbdefs.h>
 #include <dspbridge/dblldefs.h>
 
+extern bool symbols_reloaded;
+
 extern void dbll_close(struct dbll_library_obj *lib);
 extern dsp_status dbll_create(struct dbll_tar_obj **target_obj,
 			      struct dbll_attrs *pattrs);
@@ -53,4 +55,5 @@ extern dsp_status dbll_unload_sect(struct dbll_library_obj *lib,
 				   char *sectName, struct dbll_attrs *attrs);
 bool dbll_find_dsp_symbol(struct dbll_library_obj *zl_lib, u32 address,
 		u32 offset_range, u32 *sym_addr_output, char *name_output);
+
 #endif /* DBLL_ */
diff --git a/arch/arm/plat-omap/include/dspbridge/dmm.h b/arch/arm/plat-omap/include/dspbridge/dmm.h
index c8f6549..740af6b 100644
--- a/arch/arm/plat-omap/include/dspbridge/dmm.h
+++ b/arch/arm/plat-omap/include/dspbridge/dmm.h
@@ -67,4 +67,9 @@ extern void dmm_exit(void);
 
 extern dsp_status dmm_create_tables(struct dmm_object *dmm_mgr,
 				    u32 addr, u32 size);
+
+#ifdef DSP_DMM_DEBUG
+u32 dmm_mem_map_dump(struct dmm_object *dmm_mgr);
+#endif
+
 #endif /* DMM_ */
diff --git a/arch/arm/plat-omap/include/dspbridge/io_sm.h b/arch/arm/plat-omap/include/dspbridge/io_sm.h
index 3db55e0..a3c996d 100644
--- a/arch/arm/plat-omap/include/dspbridge/io_sm.h
+++ b/arch/arm/plat-omap/include/dspbridge/io_sm.h
@@ -41,6 +41,13 @@
 #define IO_SET_LONG(pContext, type, base, field, value)  (base->field = value)
 #define IO_GET_LONG(pContext, type, base, field)         (base->field)
 
+#ifdef CONFIG_BRIDGE_DVFS
+/* The maximum number of OPPs that are supported */
+extern s32 dsp_max_opps;
+/* The Vdd1 opp table information */
+extern u32 vdd1_dsp_freq[6][4];
+#endif
+
 /*
  *  ======== io_cancel_chnl ========
  *  Purpose:
@@ -295,5 +302,8 @@ dsp_status dump_dsp_stack(struct wmd_dev_context *wmd_context);
 
 void dump_dl_modules(struct wmd_dev_context *wmd_context);
 
+#ifndef DSP_TRACEBUF_DISABLED
+void print_dsp_debug_trace(struct io_mgr *hio_mgr);
+#endif
 
 #endif /* IOSM_ */
diff --git a/arch/arm/plat-omap/include/dspbridge/node.h b/arch/arm/plat-omap/include/dspbridge/node.h
index 7d228aa..a8078fb 100644
--- a/arch/arm/plat-omap/include/dspbridge/node.h
+++ b/arch/arm/plat-omap/include/dspbridge/node.h
@@ -581,4 +581,7 @@ extern dsp_status node_get_uuid_props(void *hprocessor,
 dsp_status node_find_addr(struct node_mgr *node_mgr, u32 sym_addr,
 				u32 offset_range, void *sym_addr_output,
 				char *sym_name);
+
+enum node_state node_get_state(bhandle hnode);
+
 #endif /* NODE_ */
diff --git a/arch/arm/plat-omap/include/dspbridge/proc.h b/arch/arm/plat-omap/include/dspbridge/proc.h
index 053aa22..18b51c5 100644
--- a/arch/arm/plat-omap/include/dspbridge/proc.h
+++ b/arch/arm/plat-omap/include/dspbridge/proc.h
@@ -23,6 +23,8 @@
 #include <dspbridge/devdefs.h>
 #include <dspbridge/drv.h>
 
+extern char *iva_img;
+
 /*
  *  ======== proc_attach ========
  *  Purpose:
diff --git a/drivers/dsp/bridge/pmgr/dbll.c b/drivers/dsp/bridge/pmgr/dbll.c
index 70dbba3..7f551e5 100644
--- a/drivers/dsp/bridge/pmgr/dbll.c
+++ b/drivers/dsp/bridge/pmgr/dbll.c
@@ -137,7 +137,6 @@ struct dbll_symbol {
 	struct dbll_sym_val value;
 	char *name;
 };
-extern bool symbols_reloaded;
 
 static void dof_close(struct dbll_library_obj *zl_lib);
 static dsp_status dof_open(struct dbll_library_obj *zl_lib);
diff --git a/drivers/dsp/bridge/pmgr/dmm.c b/drivers/dsp/bridge/pmgr/dmm.c
index 514b2c0..5ce531f 100644
--- a/drivers/dsp/bridge/pmgr/dmm.c
+++ b/drivers/dsp/bridge/pmgr/dmm.c
@@ -75,9 +75,6 @@ static u32 table_size;		/* The size of virt and phys pages tables */
 static struct map_page *get_region(u32 addr);
 static struct map_page *get_free_region(u32 aSize);
 static struct map_page *get_mapped_region(u32 aAddr);
-#ifdef DSP_DMM_DEBUG
-u32 dmm_mem_map_dump(struct dmm_object *dmm_mgr);
-#endif
 
 /*  ======== dmm_create_tables ========
  *  Purpose:
diff --git a/drivers/dsp/bridge/rmgr/drv.c b/drivers/dsp/bridge/rmgr/drv.c
index aab6416..69fba49 100644
--- a/drivers/dsp/bridge/rmgr/drv.c
+++ b/drivers/dsp/bridge/rmgr/drv.c
@@ -75,7 +75,6 @@ static dsp_status request_bridge_resources(struct cfg_hostres *res);
 /* GPP PROCESS CLEANUP CODE */
 
 static dsp_status drv_proc_free_node_res(bhandle hPCtxt);
-extern enum node_state node_get_state(bhandle hnode);
 
 /* Allocate and add a node resource element
 * This function is called from .Node_Allocate. */
diff --git a/drivers/dsp/bridge/rmgr/node.c b/drivers/dsp/bridge/rmgr/node.c
index a889b24..0edc7da 100644
--- a/drivers/dsp/bridge/rmgr/node.c
+++ b/drivers/dsp/bridge/rmgr/node.c
@@ -58,6 +58,7 @@
 /*  ----------------------------------- This */
 #include <dspbridge/nodepriv.h>
 #include <dspbridge/node.h>
+#include <dspbridge/dmm.h>
 
 /* Static/Dynamic Loader includes */
 #include <dspbridge/dbll.h>
@@ -259,10 +260,6 @@ static u32 ovly(void *priv_ref, u32 ulDspRunAddr, u32 ulDspLoadAddr,
 static u32 mem_write(void *priv_ref, u32 ulDspAddr, void *pbuf,
 		     u32 ul_num_bytes, u32 nMemSpace);
 
-#ifdef DSP_DMM_DEBUG
-extern u32 dmm_mem_map_dump(struct dmm_object *dmm_mgr);
-#endif
-
 static u32 refs;		/* module reference count */
 
 /* Dynamic loader functions. */
diff --git a/drivers/dsp/bridge/rmgr/proc.c b/drivers/dsp/bridge/rmgr/proc.c
index c163829..9649016 100644
--- a/drivers/dsp/bridge/rmgr/proc.c
+++ b/drivers/dsp/bridge/rmgr/proc.c
@@ -74,8 +74,6 @@
 #define RBUF		0x4000		/* Input buffer */
 #define WBUF		0x8000		/* Output Buffer */
 
-extern char *iva_img;
-
 /*  ----------------------------------- Globals */
 
 /* The proc_object structure. */
diff --git a/drivers/dsp/bridge/wmd/_tiomap.h b/drivers/dsp/bridge/wmd/_tiomap.h
index 56efb84..81f61fa 100644
--- a/drivers/dsp/bridge/wmd/_tiomap.h
+++ b/drivers/dsp/bridge/wmd/_tiomap.h
@@ -351,6 +351,12 @@ struct wmd_dev_context {
 	u32 dsp_per_clks;
 };
 
+/*
+ * If dsp_debug is true, do not branch to the DSP entry
+ * point and wait for DSP to boot.
+ */
+extern s32 dsp_debug;
+
 	/*
 	 * ======== wmd_tlb_dsp_va_to_mpu_pa ========
 	 *     Given a DSP virtual address, traverse the page table and return
diff --git a/drivers/dsp/bridge/wmd/_tiomap_pwr.h b/drivers/dsp/bridge/wmd/_tiomap_pwr.h
index 1948e83..5212101 100644
--- a/drivers/dsp/bridge/wmd/_tiomap_pwr.h
+++ b/drivers/dsp/bridge/wmd/_tiomap_pwr.h
@@ -19,6 +19,12 @@
 #ifndef _TIOMAP_PWR_
 #define _TIOMAP_PWR_
 
+#ifdef CONFIG_PM
+extern s32 dsp_test_sleepstate;
+#endif
+
+extern struct mailbox_context mboxsetting;
+
 /*
  * ======== wake_dsp =========
  * Wakes up the DSP from DeepSleep
diff --git a/drivers/dsp/bridge/wmd/io_sm.c b/drivers/dsp/bridge/wmd/io_sm.c
index d815195..4fcd9f6 100644
--- a/drivers/dsp/bridge/wmd/io_sm.c
+++ b/drivers/dsp/bridge/wmd/io_sm.c
@@ -151,22 +151,11 @@ static u32 read_data(struct wmd_dev_context *hDevContext, void *dest,
 static u32 write_data(struct wmd_dev_context *hDevContext, void *dest,
 		      void *pSrc, u32 usize);
 
-#ifndef DSP_TRACEBUF_DISABLED
-void print_dsp_debug_trace(struct io_mgr *hio_mgr);
-#endif
-
 /* Bus Addr (cached kernel) */
 static dsp_status register_shm_segs(struct io_mgr *hio_mgr,
 				    struct cod_manager *cod_man,
 				    u32 dw_gpp_base_pa);
 
-#ifdef CONFIG_BRIDGE_DVFS
-/* The maximum number of OPPs that are supported */
-extern s32 dsp_max_opps;
-/* The Vdd1 opp table information */
-extern u32 vdd1_dsp_freq[6][4];
-#endif
-
 /*
  *  ======== bridge_io_create ========
  *      Create an IO manager object.
diff --git a/drivers/dsp/bridge/wmd/mmu_fault.h b/drivers/dsp/bridge/wmd/mmu_fault.h
index 537e6e7..74db489 100644
--- a/drivers/dsp/bridge/wmd/mmu_fault.h
+++ b/drivers/dsp/bridge/wmd/mmu_fault.h
@@ -19,6 +19,8 @@
 #ifndef MMU_FAULT_
 #define MMU_FAULT_
 
+extern u32 fault_addr;
+
 /*
  *  ======== mmu_fault_dpc ========
  *      Deferred procedure call to handle DSP MMU fault.
diff --git a/drivers/dsp/bridge/wmd/tiomap3430.c b/drivers/dsp/bridge/wmd/tiomap3430.c
index 4ac53ea..22f4d76 100644
--- a/drivers/dsp/bridge/wmd/tiomap3430.c
+++ b/drivers/dsp/bridge/wmd/tiomap3430.c
@@ -160,12 +160,6 @@ struct pg_table_attrs {
 };
 
 /*
- *  If dsp_debug is true, do not branch to the DSP entry point and wait for DSP
- *  to boot
- */
-extern s32 dsp_debug;
-
-/*
  *  This mini driver's function interface table.
  */
 static struct bridge_drv_interface drv_interface_fxns = {
diff --git a/drivers/dsp/bridge/wmd/tiomap3430_pwr.c b/drivers/dsp/bridge/wmd/tiomap3430_pwr.c
index 3163d1d..86a6cc0 100644
--- a/drivers/dsp/bridge/wmd/tiomap3430_pwr.c
+++ b/drivers/dsp/bridge/wmd/tiomap3430_pwr.c
@@ -45,11 +45,6 @@
 #include <mach-omap2/prm-regbits-34xx.h>
 #include <mach-omap2/cm-regbits-34xx.h>
 
-#ifdef CONFIG_PM
-extern s32 dsp_test_sleepstate;
-#endif
-extern struct mailbox_context mboxsetting;
-
 /*
  *  ======== handle_constraints_set ========
  *  	Sets new DSP constraint
diff --git a/drivers/dsp/bridge/wmd/ue_deh.c b/drivers/dsp/bridge/wmd/ue_deh.c
index 55a6790..d6cd820 100644
--- a/drivers/dsp/bridge/wmd/ue_deh.c
+++ b/drivers/dsp/bridge/wmd/ue_deh.c
@@ -193,7 +193,6 @@ void bridge_deh_notify(struct deh_mgr *deh_mgr, u32 ulEventMask, u32 dwErrInfo)
 	dsp_status status = DSP_SOK;
 	u32 mem_physical = 0;
 	u32 hw_mmu_max_tlb_count = 31;
-	extern u32 fault_addr;
 	struct cfg_hostres *resources;
 	hw_status hw_status_obj;
 	u32 cnt = 0;
-- 
1.7.0.4

